jekyll/_cci2/migrating-from-github.adoc

```diff
@@ -0,0 +1,278 @@
+= Migrating from Github Actions
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+:sectanchors:
+
+This document provides an overview of how to migrate from Github Actions to CircleCI.
+
+== Why Migrate From Github Actions to CircleCI?
+
+CircleCI is a first-class CI tool. CI/CD is what we specialize in, and we have been doing it for over 8 years since the company's founding. On top of normal features you'd expect from any CI/CD tool, what sets us apart from others are the various additional features that boost productivity:
```

Comment:
```suggestion
CircleCI is a first-class CI tool. CI/CD has been our specialization since the company's founding 8 years ago. On top of the features you would expect from any CI/CD tool, what sets us apart are the following productivity-boosting features:
```